src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -472,19 +475,19 @@ public static void cloneFromScm(
      *   <li><code>git checkout FETCH_HEAD</code>
      * </ul>
      *
-     * @param connectionURL The connection URL, in a format such as
-     *     scm:git:https://github.com/jenkinsci/mailer-plugin.git or
-     *     https://github.com/jenkinsci/mailer-plugin.git
+     * @param gitURL The git native URL, see the <a
```

Comment:
makes the logging easier to follow if it spits out the URL that is actually going to be used for the `git fetch`